lsp/server/aws-lsp-codewhisperer/src/client/codewhispererclient.d.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * THIS FILE IS AUTOGENERATED BY 'generateServiceClient.ts'.
```

Comment:
👀

Reply:
I used the AWS Toolkit for VS Code repo to produce this client. I didn't set anything up in this repo (its not relevant for this proof of life phase).